src/devcards/core.clj

```diff
@@ -178,13 +179,18 @@
   (when (utils/devcards-active?)
     `(devcards.util.utils/pprint-code ~obj)))
 
-(defmacro mkdn-code [body] `(str "\n```\n" ~body "```\n"))
+(defmacro mkdn-code [body] `(str "\n```clojure\n" ~body "\n```\n"))
```

Comment:
I couldn't figure out how the existing code is displaying properly with the use of the `clojure` hint in the markdown, so I added it here. But it makes me think that something is going on with the markdown handling that I don't fully understand. Also, if I didn't add the extra newline then I was seeing three trailing singlequotes as part of the code display. Again, not sure why this only effects the macro that I added. Seems worth investigating/understanding.